packages/zoe/src/contracts/operaConcertTicket.js

```diff
@@ -0,0 +1,120 @@
+import { makeZoeHelpers, defaultAcceptanceMsg } from './helpers/zoeHelpers';
+
+import produceIssuer from '@agoric/ertp';
+
+/*
+  Roles in the arrangement:
+  - Contract creator: describes the contract with:
+    - number of seats, show, date/time of start
+    - expected (ERTP) amount per ticket (we assume all tickets cost the same)
+  - Smart Contract: 
+    - mints the tickets
+    - provides the seats
+  - Auditorium (unique contract seat, usually taken by the contract creator): the person hosting 
+  the Opera show, selling the tickets and getting the payment back
+  - Ticket buyers (contract seat created on demand): 
+    - can see the available opera show seats
+    - can consult the terms
+    - can redeem the zoe invite with the proper payment to get the ticket back
+*/
+
+export const makeContract = harden(async zoe => {
+
+  const zoeService = zoe.getZoeService();
+
+  // Create the internal ticket mint
+  const { issuer, mint, amountMath } = produceIssuer('Opera tickets', 'set');
+  await zoe.addNewIssuer(issuer, 'Auditorium');
```

Comment:
I think Mark would prefer that we not use `async/await` in our examples.

Reply:
i didn't know that. I'll fix it
Would it be worth adding an eslint warning (which is different than an eslint error) for this purpose?

Reply:
> Would it be worth adding an eslint warning (which is different than an eslint error) for this purpose? [avoiding `await`]

Yes. We should start with a warning.